be/src/formats/orc/apache-orc/c++/src/Reader.cc

```diff
@@ -594,6 +598,13 @@ std::unique_ptr<StripeInformation> ReaderImpl::getStripe(uint64_t stripeIndex) c
             contents->blockSize, contents->readerMetrics));
 }
 
+const orc::proto::StripeInformation& ReaderImpl::getStripeInOrcFormat(uint64_t stripeIndex) const {
+    if (stripeIndex > getNumberOfStripes()) {
+        throw std::logic_error("stripe index out of range");
```

Comment:
We didn't catch the exception in HdfsOrcScanner, it may occur BE crashed.

Reply:
It's just code convention of apache-orc.